src/lib/util/Utils.ts

```diff
@@ -6,4 +6,11 @@ const deepCopy = (obj: any): any => {
 	return JSON.parse(JSON.stringify(obj));
 }
 
-export { deepCopy };
+const stringArrayTypeGuard = (object): object is string[] => {
```

Comment:
This new utility function gets used in various places to let the compiler know that something is definitely a string array.